pkg/util/taints/taints.go

```diff
@@ -20,10 +20,22 @@ package taints
 import (
 	"fmt"
 	"strings"
+	"time"
 
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation"
+
 	"k8s.io/kubernetes/pkg/api"
```

Comment:
> Any idea where? API helpers doesn't depend on client, so it doesn't fit there as well. It's only needed because of those wonderful annotations...

Are the only callers in tests? If so, tests make sense.  For controller related ones, its controller package could work.